src/app/reservation/create/create.component.ts

```diff
@@ -13,7 +13,7 @@ import { CreateReservationModule } from './create.module';
   styleUrl: './create.component.scss'
 })
 
-export class CreateReservationComponent {
+export class CreateReservationComponent1 {
```

Comment:
waarom heet deze CreateReservationComponent1? Het is beter zonder de 1

Reply:
dit is de component die stan gemaakt heeft. ik moest die nog verwijderen.